TeamCode/src/main/java/org/firstinspires/ftc/teamcode/drive/SampleMecanumDrive.java

```diff
@@ -375,4 +372,15 @@ public void setMotorPowers(double v, double v1, double v2, double v3) {
     public double getRawExternalHeading() {
         return imu.getAngularOrientation().firstAngle;
     }
+
+    double getBatteryVoltage(HardwareMap hardwareMap) {
+        double result = Double.POSITIVE_INFINITY;
```

Comment:
I don't think returning `Double.POSITIVE_INFINITY` as a sentinel is a good idea. I would personally implement this with the hw map iterator idiom (`hardwareMap.voltageSensor.iterator().next().getVoltage()`) which has the benefit of throwing an exception when no voltage sensors are present. Of course, this is all somewhat paranoid.